public/autocomplete/accessible-autocomplete.min.js

```diff
@@ -0,0 +1,1115 @@
+(function webpackUniversalModuleDefinition(e, t) {
```

Comment:
Looks complex. Was this simply a grab and drop of the standard JS?

Reply:
this is part of the component itself, I've just imported it :)